semantic/src/site/globals/site.overrides

```diff
@@ -1206,6 +1215,8 @@ a:hover {
   margin-top: -10px;
   margin-bottom: -10px;
   height: 60px;
+  flex-wrap: nowrap;
+  font-size: @slightlySmallerText;
```

Comment:
Made Progress Bar text slightly smaller, and also prevented row wrapping, as the "Grid" component was wrapping at a certain medium width, which looked wonk.